clients/web/src/views/layout/FooterView.js

```diff
@@ -9,9 +9,18 @@ import 'girder/stylesheets/layout/footer.styl';
  * This view shows the footer in the layout.
  */
 var LayoutFooterView = View.extend({
+
+    initialize: function (settings) {
+        const mail = settings.contactEmail || null;
+        if (mail !== null) {
+            this.emailHref = `mailto: ${mail}`;
```

Comment:
When you propose to convert into JSON you mean in the webroot.py ? Just like that : 
`contactEmail = json.loads(ModelImporter.model('setting').get(SettingKey.CONTACT_EMAIL_ADDRESS))`

And I just insert this into the template webroot.mako ?